ci/properties/jekyll-docker.properties.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Jekyll",
+    "name": "Jekyll using Docker image",
```

Comment:
I think `Jekyll  container` will be better name as workflow is packaging Jekyll into container.

Reply:
I wanted to be consistent with the filename. As it includes `docker`